seatunnel-plugin-discovery/src/main/java/org/apache/seatunnel/plugin/discovery/AbstractPluginDiscovery.java

```diff
@@ -427,20 +429,12 @@ public boolean accept(File pathname) {
                                                         pathname.getName(), pluginJarPrefix);
                                     }
                                 });
-        if (ArrayUtils.isEmpty(targetPluginFiles)) {
+        if (ArrayUtils.isEmpty(targetPluginFiles) || targetPluginFiles == null) {
```

Comment:
```suggestion
        if (ArrayUtils.isEmpty(targetPluginFiles)) {
```
`ArrayUtils.isEmpty` already contains null check.

Reply:
yes, misled by idea editor